Meta/run.py

```diff
@@ -639,13 +656,15 @@ def set_up_boot_drive(config: Configuration):
         config.add_devices(["ahci,id=boot-drive-ahci", "ide-hd,drive=boot-drive,bus=boot-drive-ahci.0"])
         config.kernel_cmdline.append("root=ahci0:0:0")
     if config.boot_drive_type == BootDriveType.NVMe:
-        config.add_devices(
-            [
+        if config.architecture == Arch.x86_64:
+            config.add_devices([
                 "i82801b11-bridge,id=bridge4",
                 "nvme,serial=deadbeef,drive=boot-drive,bus=bridge4,logical_block_size=4096,physical_block_size=4096",
-            ]
-        )
-        config.kernel_cmdline.append("root=nvme0:1:0")
+            ])
+            config.kernel_cmdline.append("root=nvme0:1:0")
+        else:
+            config.add_devices(["nvme,serial=deadbeef,drive=boot-drive"])
```

Comment:
NVMe for some reason doesn't work in aarch64 kvm (and no other storage controller either I think).
Should we maybe only allow virtualization support on macOS hosts for now?

Reply:
Sure